app/rev2/prelaunch/mock/mocks.c

```
@@ -0,0 +1,79 @@
+return dashboard_dump_return;
+}
+
+void fc_state_update
+    (
+    FLIGHT_COMP_STATE_TYPE new_state
+    )
+{
+
+}
+
+LORA_STATUS lora_cmd_execute
```

Cppcheck (2.20.0)

[style] The function 'lora_cmd_execute' is never used.

(unusedFunction)